Directory.Build.props

```diff
@@ -191,9 +190,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <DotNetOutputPath>$(MSBuildThisFileDirectory)bin/</DotNetOutputPath>
-    <DotNetTempDirectory>$(DotNetOutputPath)temp/</DotNetTempDirectory>
-    <DotNetDirectory>$(DotNetOutputPath)dotnet/</DotNetDirectory>
+    <DotNetTempDirectory>$(RepoRoot)/temp/</DotNetTempDirectory>
```

Comment:
Where does `RepoRoot` come from and if I run a build in VS will it be a blank string making this evaluate to C:\temp?

Reply:
just tried our cake on vs and it works, this temp dir is just install the .net install script and workload packs before installing the iOS and android workloads
. 

Can you take these for a run and try the build.sh -pack command ?

Reply:
I got it to pack fine but I think I might be missing the NativeAssets thing ?